### ISSUE TITLE
Additions to make (huge spoilers so don't look!)

### ISSUE THREAD

**Issue by hearth1an** [hearth1an/hearth1an.ChristmasStory#50]
- [ ] Add nomai gravity platform to Riebeck platform (easier landing)
- [ ] Replace Prisoner near the third totem after the vision (also torch, footprints, dream world versions)
- [ ] Add socket to the hole inside the Prison 
- [ ] Fix Arkose and Moraine dialogue range
- [ ] Fix Hornfels dialogue focal point

**Comment by nice2cu1** [comment]
Is it a little too difficult to invite a Prisoner? Here I'm stuck (maybe I'm not smart.), and I'm struggling with how to unlock the third lock without dying. Eventually, after watching a YouTube promotional video where you interacted with the player, I learned how to properly unlock the third lock. Is the guidance too weak here.

**Comment by hearth1an** [comment]
_how to unlock the third lock without dying._
You'll need to die first as usual and after showing Prisoner the vision you should come up and try investigate the area. You should pay attention to details.

**Comment by nice2cu1** [comment]
Well, I thought there would be a related optimization schedule. This place stuck me all afternoon!! Ha ha!!

**Comment by hearth1an** [comment]
Yes, there will be some optimisation to this, but not sure if I'll do all stuff this week.
